source/nodejs/adaptivecards-templating/src/shared.ts

```diff
@@ -2,4 +2,6 @@
 // Licensed under the MIT License.
 export class GlobalSettings {
     static undefinedExpressionValueSubstitutionString?: string = undefined;
+    static expressionStartMarker = "${";
+    static expressionEndMarker = "}";
```

Comment:
@nesalang is your design spec updated with this new marker syntax ? Also i thought we could use the antlr based parser for JS also - no? is that a discussion item ?

Reply:
I don't have a design spec. I am literally just porting the existing template engine, which is a very small piece of code which isn't done using Antlr. The Expression parser/evaluator, e.g. Adaptive Expressions, which the templating code utilizes, is based on Antlr.

Reply:
the configurable start and end tokens for templated string is captured in the project board. I think we discussed it, and decided during the last wednesday (3/25) work group meeting that it will be JS only feature to help current JS libary user migrate to new version of JS SDK. We don't have plan for JS to use antlr for parsing templated adaptive cards. .Net Antlr parsers/lexer can be used for ios/android/uwp, and I believe that is the current plan.

---
In reply to: [398591949](https://github.com/microsoft/AdaptiveCards/pull/3867#discussion_r398591949) [](ancestors = 398591949)